### DISCUSSION TITLE
Canvas keeps crashing

### DISCUSSION THREAD

**Discussion by RuSkiess** [thatskymod/Sky-CotL-Scripts#138] [Q&A] [answered]
I recently just updated my sky to the latest update, and afterwards I opened canvas and found out it keeps crashing. 

The original sky opens just fine, I tried restarting my phone as well as reinstalling canvas and the mod I have (Dukun Langit) but it still won't open. Any suggestions on how I can get it to work? :(

I thought it might've been because I erased the cache for canvas but when I reinstalled it it still wouldn't work.

**Comment by Krullerwhit** [comment]
It's broken from the latest update. They'll likely update it soon. C:

**Comment by thatskymod** [comment, marked as answer]
Thanks for reaching out! Let's go through a few steps to ensure your setup is up-to-date.

1. ~Update Sky to version [0.27.0](https://thatgamecompany.helpshift.com/hc/en/17-sky-children-of-the-light/faq/1356/)~ (it seems you've already done this)
2. Update **Canvas** to the newly released version [1.5.6](https://github.com/thatskymod/Sky-CotL-Scripts/blob/754d38203696c8feea8dff40c01f492befc6d85f/Canvas/Canvas.apk)
3. Update **libLangit** to the—also newly released—version [0.4.4](https://github.com/thatskymod/Sky-CotL-Scripts/blob/05fa329f2e50de2d33b6232b5005705f45e5ac91/Canvas/Libs/libLangit044.so).

This combination should resolve the immediate crashing.

Detailed change lists can be found at [Langit Discussion Board](https://t.me/dukunlangit/8/40577), the Canvas [releases page](https://github.com/RomanChamelo/Canvas-Open-Source/releases/latest) and thatgamecompany's [patch notes](https://thatgamecompany.helpshift.com/hc/en/17-sky-children-of-the-light/faq/1356/).

**Reply by RuSkiess** [reply]
Thank you ^^ I'll try it once I get the time.
